crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -435,6 +435,14 @@ define_tedge_config! {
     },
 
     mqtt: {
+        /// MQTT topic root
```

Comment:
This comment goes directly to the description showed by `tedge config --doc`. So I want to have a bit better description. (Another suggestion is welcomed!)
```suggestion
        /// The MQTT root/base topic prefix
```